### DISCUSSION TITLE
HexLsbAsciiToUint64()

### DISCUSSION THREAD

**Discussion by FalconBry** [Makuna/Rfid134#15] [Q&A] [answered]
Hello, Thank you for your rfid code.
But where can I find the implementation of the HexLsbAsciiToUint64() function?
Is it in some Arduino library?

Thank you.

**Comment by Makuna** [comment, marked as answer]
It's part of this library.  It is in the header file you include in your sketch.

**Reply by FalconBry** [reply]
Sorry, I must be blind!
Thank you.
